nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractPutElasticsearch.java

```diff
@@ -165,7 +165,7 @@ void logElasticsearchDocumentErrors(final IndexOperationResponse response) throw
 
             if (logErrors) {
                 getLogger().error(output);
-            } else {
+            } else if (getLogger().isDebugEnabled()) {
```

Comment:
Aside from the conditional `isDebugEnabled()` in the service implementation that logs the HTTP entity, this is the only other conditional that seems necessary.

Reply:
Ah, yeah, forgot to follow up on that form my original comment (https://github.com/apache/nifi/pull/6712#pullrequestreview-1192230945), good catch